src/utils/iqeEnablement.ts

```diff
@@ -11,6 +11,8 @@ import type { AuthContextProps } from 'react-oidc-context';
 let xhrResults: XMLHttpRequest[] = [];
 let fetchResults: Record<string, unknown> = {};
 
+// this extra header helps with API metrics
+const FE_ORIGIN_HEADER_NAME = 'x-rh-frontend-origin';
```

Comment:
This will effect FedRamp

Reply:
What is the effect? Is there only a set of allowed headers?

Reply:
I tested this new header on our pre-prod env and haven't seen any issues. Since I'm aware this change is being made I'll keep an eye out for any issues and can follow up with a gate if required.